src/tools/perf/lib/libperf.c

```diff
@@ -336,6 +336,15 @@ static ucs_status_t ucx_perf_test_check_params(ucx_perf_params_t *params)
     return UCS_OK;
 }
 
+void uct_perf_ep_fence(ucx_perf_context_t *perf, int peer_index)
+{
+    uct_ep_h ep = perf->uct.peers[peer_index].ep;
```

Comment:
maybe remove ep var?
moreover, I'm not sure we need this wrapper, it's used only once, can call uct_ep_fence directly from uct_perf_test_runner::fence